project/src/components/raiting-input/raiting-input.tsx

```diff
@@ -0,0 +1,16 @@
+import {ChangeEvent} from 'react';
+
+type RaitingInputProps = {
+  id: number;
+  raitingFilm: number;
+  handleInputChange: ({target}:ChangeEvent<HTMLInputElement>) => void;
```

Comment:
```suggestion
  handleInputChange: ({target}: ChangeEvent<HTMLInputElement>) => void;
```

клавиатуру смени, пробел сломался))